dav2rem.py

```diff
@@ -24,7 +24,6 @@
 from netrc import netrc
 from os.path import basename, expanduser, splitext
 from remind import Remind
-from urlparse import urlparse
```

Comment:
I guess you want `from urllib.parse import urlparse` here.

Reply:
No. It works fine without import.